config/routes.rb

```diff
@@ -70,6 +70,7 @@
         delete :remove_admin
       end
     end
+    resources :profile_fields, only: %i[index update create destroy]
```

Comment:
**suggestion (non-blocking):** You could use the shorter `except` version here.

```suggestion
    resources :profile_fields, except: %i[new edit show]
```

Reply:
@citizen428 this is the only suggestion I didn't implement. I feel that being more verbose (instead of using the negation) is beneficial here.

Reply:
No problem. There are 7 actions, so I usually use whichever form leads to less than 4 in the array. But I know many people prefer to list the available actions instead.